packages/build/src/npm-packages/publish.ts

```diff
@@ -20,10 +20,6 @@ export function publishNpmPackages(
     );
   }
 
-  if (versions[0] === PLACEHOLDER_VERSION) {
-    throw new Error('Refusing to publish packages with placeholder version');
-  }
```

Comment:
I guess we're dropping this because we're assuming that a placeholder version wouldn't ever be used anyway?

Reply:
that was my thinking though I am wondering if it'd be useful in some manner; the only thing I can think of is if we want to opt-out a package from being versioned independently